File: src/Services/Announcement/Announcement.Common/Models/Employee.cs
using System.Collections.Generic;

namespace Announcement.Common.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Initials { get; set; }

        public List<Announcement> Announcements { get; set; }
    }
}

File: src/Web/Spa/src/app/models/employee.ts
import { School } from './school';
import { Room } from './room';
import { Building } from './building';

export class Employee{
    id:number;
    pcn:string;
    firstName:string;
    lastName:string;
    initials:string;
    email:string;
    phoneNumber:string;
    accountStatus:string;
    school:School;
    room:Room;
    building:Building;
}

export class ShortEmployee{
    id:number;
    firstName:string;
    lastName:string;
    initials:string;
}